h_program-lang/ast_generic.ml

```diff
@@ -896,7 +896,7 @@ and macro_definition = {
 and directive = 
   (* newvar: *)
   | ImportFrom of tok (* 'import'/'from' for Python, 'include' for C *) * 
-                  module_name * alias list
+                  module_name * alias option
```

Comment:
I don't think it should be an option here. 
If you import just a module, you should use the ImportAs construct
If you import an entity, it should be ImportFrom, but there will always be an entity (the first element in the alias type),
what is optional is whether you want to alias this first element (hence the option in the second element of the alias type).

Reply:
In fact in the rest of the diff you always use Some (...) so it's never a None here.

Reply:
maybe alias is the wrong name for this type ... it's really ident_and_possibly_alias